app/atoms/base/_icomoon.scss

```diff
@@ -0,0 +1,31 @@
+@font-face {
+  font-family: "icomoon";
+  src:  url("../fonts/icomoon/icomoon.eot?9rc3go");
+  src:  url("../fonts/icomoon/icomoon.eot?9rc3go#iefix") format("embedded-opentype"), url("../fonts/icomoon/icomoon.ttf?9rc3go") format("truetype"), url("../fonts/icomoon/icomoon.woff?9rc3go") format("woff"), url("../fonts/icomoon/icomoon.svg?9rc3go#icomoon") format("svg");
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: "icomoon";
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+// Assign classes to the icons
+// (you can delete the example below)
+
+.icon-up-arrow:before {
+  content: $icon_arrow-up;
```

Comment:
Ok makes sense I think if this is an example you should added as a document above and not as code since is not going to be used.